packages/amplify-category-geo/src/provider-controllers/import.ts

```diff
@@ -0,0 +1,133 @@
+import { readFileSync } from 'fs-extra';
+import { $TSContext } from 'amplify-cli-core';
+import { prompter, printer } from 'amplify-prompts';
+import { existsSync, writeFileSync } from 'fs-extra';
+import { join } from 'path';
+import aws from 'aws-sdk';
+import { ServiceName } from '../service-utils/constants';
+import { validateGeoJSONObj } from '../service-utils/validateGeoJSONObj';
+import {
+  FeatureCollection, ImportParams, GeofenceCollectionParams, GeofenceParams, IdentifierOption, IdentifierInfo,
+} from '../service-utils/importParams';
+
+const MAX_ENTRIES_PER_BATCH = 10;
+
+export const importResource = async (context: $TSContext) => {
+  const geofenceCollectionResources = ((await context.amplify.getResourceStatus()).allResources as any[])
```

Comment:
can we get this info from amplify meta? If its too much refactor, at-least add a TODO to avoid using `getResourceStatus`